### ISSUE TITLE
Can't build from sdist with pip >= 19

### ISSUE THREAD

**Issue by atilag** [Qiskit/qiskit-aer#193]
### Informations

- **Qiskit Aer version**: 0.2.0
- **Python version**: All with pip >= 19
- **Operating system**: All

### What is the current behavior?
If we have pip 19 installed, installing from sdist (via pip install) will fail with an error like:
```
...
Could NOT find Cython (missing: CYTHON_EXECUTABLE)
...
```

### Suggested solutions
Donwgrade to pip 18 will solve this problem.
I have filed a new issue reporting this problem here: https://github.com/pypa/pip/issues/6360

**Comment by atilag** [comment]
Installing with the `--no-binary` option of pip will rise this error:
```
File "/home/jgomez/anaconda3/envs/test1/bin/cython", line 6, in <module>
    from Cython.Compiler.Main import setuptools_main

    ModuleNotFoundError: No module named 'Cython'
```

**Comment by atilag** [comment]
This is fixed as per `pip 19.3.1' :)
